### PR TITLE
Wangjun referencecommon

### PR DESCRIPTION
**Pull request by wangjun-microsoft**
# Please do not merge this PR until the deployment of new version of Service as the changes are not compatible with previous version.

## TODOLIST
- [x] Updatethe version in `packages.config` of Sample.
- [x] Update the `Reference` in Sample `csproj`.
- [x] Upload new version of nuget package.
- [x] Update the `README.md`.
- [ ] Clean related issues.
- [ ] Publish a new release on github.

### CODE REVIEW

File: Sample-WPF/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -80,5 +80,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
\ No newline at end of file
+[assembly: AssemblyVersion("1.3.0")]
+[assembly: AssemblyFileVersion("1.3.0")]
```
Comment (wangjun-microsoft) [inline Sample-WPF/Properties/AssemblyInfo.cs]: update to 1.2.5

File: Sample-WPF/Controls/FaceDetectionPage.xaml [hunk 2]
```diff
@@ -126,10 +126,10 @@
                             </Image>
                             <TextBlock Grid.Column="2" Grid.Row="0" HorizontalAlignment="Left" VerticalAlignment="Top" Text="{Binding Gender}" />
                             <TextBlock Grid.Column="2" Grid.Row="0" HorizontalAlignment="Right" VerticalAlignment="Top" Text="{Binding Age}" />
-                            <TextBlock Grid.Column="2" Grid.Row="1" HorizontalAlignment="Left" VerticalAlignment="Top" Text="{Binding IsSmiling}" />
```
Comment (wangjun-microsoft) [inline Sample-WPF/Controls/FaceDetectionPage.xaml]: add blank spaces before the commands

Comment (wangjun-microsoft) [inline Sample-WPF/Controls/FaceDetectionPage.xaml]: done.

File: Sample-WPF/Controls/FaceDetectionPage.xaml.cs [hunk 3]
```diff
@@ -230,7 +227,7 @@ public string SelectedFile
                         string subscriptionKey = mainWindow._scenariosControl.SubscriptionKey;
 
                         var faceServiceClient = new FaceServiceClient(subscriptionKey);
-                        Contract.Face[] faces = await faceServiceClient.DetectAsync(fileStream, false, true, new FaceAttributeType[] { FaceAttributeType.Gender, FaceAttributeType.Age, FaceAttributeType.Smile, FaceAttributeType.Glasses, FaceAttributeType.HeadPose, FaceAttributeType.FacialHair });
+                        Contract.Face[] faces = await faceServiceClient.DetectAsync(fileStream, false, true, new FaceAttributeType[] { FaceAttributeType.Gender, FaceAttributeType.Age, FaceAttributeType.Smile, FaceAttributeType.Glasses, FaceAttributeType.HeadPose, FaceAttributeType.FacialHair, FaceAttributeType.Emotion });
```
Comment (huxuan) [inline Sample-WPF/Controls/FaceDetectionPage.xaml.cs]: We can remove the `smile` attribute request now.

**Pull request merged**